include/swift/AST/Decl.h

```diff
@@ -351,7 +351,11 @@ class alignas(1 << DeclAlignInBits) Decl {
 
     /// \brief Whether this is a property used in expressions in the debugger.
     /// It is up to the debugger to instruct SIL how to access this variable.
-    IsDebuggerVar : 1
+    IsDebuggerVar : 1,
+
+    /// \brief Whether this is a property defined in the debugger's REPL.
```

Comment:
Sort of besides the point because it's just copied form the line above, but the `\brief` is redundant :-)